<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>709</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>串口命令行</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_UI">
    <attribute name="title">
     <string>串口命令行</string>
    </attribute>
    <widget class="QLineEdit" name="tLine_message">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>640</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MessageWindow:</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="tEdit_shell">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1061</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>640</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>640</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>扫描串口/F5</string>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>640</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayout_Baudlist">
      <item>
       <widget class="QLabel" name="label_Baudlist">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>波特率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_Baudlist">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>0</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>122</red>
              <green>122</green>
              <blue>122</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>640</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayout_Portlist">
      <item>
       <widget class="QLabel" name="label_Portlist">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>串口列表:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_Portlist">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pButton_send">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>640</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送/Enter</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>1041</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Set">
    <attribute name="title">
     <string>设置</string>
    </attribute>
    <widget class="QLabel" name="lable_quickCompelatList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快捷补全列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pButton_AddquickCompelat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>应用</string>
     </property>
     <property name="shortcut">
      <string extracomment="应用快速补全Alt+S ">Alt+S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pButton_delquickCompelat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>520</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除当前</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_autoConect">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>单个串口自动以默认波特率连接</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plaintEdit_quickCompleat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>241</width>
       <height>301</height>
      </rect>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_caseSensitive">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>大小写敏感</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>51</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_colorlist">
        <property name="text">
         <string>字体颜色:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_ColorList">
        <item>
         <property name="text">
          <string>RED</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WHITE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BLUE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GREEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YELLOW</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>117</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>默认波特率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_autoBaud">
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pButton_send</tabstop>
  <tabstop>tEdit_shell</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>cBox_Portlist</tabstop>
  <tabstop>cBox_Baudlist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
